components/PopUp.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { FunctionComponent } from "react";
```

Comment:
I know we made this as a placeholder. Now that we have a Modal and a Panel, so let's do something similar. Checkout how we are using those in `components/Board/PlayArea/ActionMenu.tsx` for the pause menu. Maybe also rename this to ConfirmationPopup instead? Since Modal is also a popup.